README.md

```diff
@@ -44,8 +42,7 @@ Run a commit with `openSearcher=true` on your `quickstart` collection to make th
 
 ## More Information
 
-The Lucille project is developed and maintained by KMW Technology ([kmwllc.com](https://kmwllc.com/)). 
+The Lucille project is developed and maintained by KMW Technology ([kmwllc.com](https://kmwllc.com/)).
 For more information regarding Lucille, please [contact us](https://kmwllc.com/index.php/contact-us/).
 
-
-
+For documentation, setup information, and configuration reference, please visit the Lucille [documentation site](https://kmwtechnology.github.io/lucille/docs/introduction/).
```

Comment:
This link is fine for now but we should change this to the landing page of the doc site when that page is usable.